### DISCUSSION TITLE
Just a talk

### DISCUSSION THREAD

**Discussion by comdar16** [Pryaxis/TShock#2474] [General]
I recently was looking for plugins and i was thinking why so much abonded/un-updated plugins? Isn't tshock is well known mod in Terraria. Like there so much good abonded stuff on old forums and some on new plugin topic.

**Comment by comdar16** [comment]
Like its not bad but it is just weird. There are many people working on TShock but no one wants to update these old plugins.

**Comment by bartico6** [comment]
People move on with their lives and leave their plugins behind. The scene is small and competitive, so when servers *do* update these plugins, they usually keep them to themselves (or rewrite them altogether).  
  
There are people who bother updating plugins and pushing them to github, but that's their kindness and interest, not something that should be taken for granted.

**Reply by comdar16** [reply]
yea but it is kinda sad at some point because there are no much plugin creators for the servers.
